### PR TITLE
Adds required plugin namespace

### PR DESCRIPTION
**Pull request by mothershiprobbie**
Breaking change introduced in AGP 8 https://developer.android.com/build/releases/past-releases/agp-8-0-0-release-notes#namespace-dsl

### CODE REVIEW

**Comment by d-nazarov** [comment]
Hi @roland-misica 
Is it possible to know what are the expected timelines of the flutter SDK update? It hasn't been updated for 3 months already. 
This issue is causing a major problem, since we're using other SDKs in our application that require Graddle 8+, but we can't build the project with it, since the Bloomreach SDK is extremely outdated and doesn't support it. This blocks our development process. 
So basically, we need to understand if we expect the SDK update any time soon or the only option is to fork the SDK, make the required changes ourselves and use it instead of the official SDK.
Thank you in advance!

**Comment by xcibik00** [comment]
Hi @d-nazarov , we are currently working on new Flutter version, but the timeline/release date is not set yet.
We will keep you updated

**Comment by mothershiprobbie** [comment]
> Hi @roland-misica Is it possible to know what are the expected timelines of the flutter SDK update? It hasn't been updated for 3 months already. This issue is causing a major problem, since we're using other SDKs in our application that require Graddle 8+, but we can't build the project with it, since the Bloomreach SDK is extremely outdated and doesn't support it. This blocks our development process. So basically, we need to understand if we expect the SDK update any time soon or the only option is to fork the SDK, make the required changes ourselves and use it instead of the official SDK. Thank you in advance!

I had the same problem, hence the fork. "the timeline/release date is not set yet" is a bit rough when it's literally stopping a build... @d-nazarov you can use my fork for now or simply add the name space

**Comment by xcibik00** [comment]
Hi all, just now we added support for Gradle 8 in 2.0.0 SDK version
